File: 2727-number-of-senior-citizens/number-of-senior-citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) 
    {
        int count = 0;
        for(int i = 0;i < details.size();i++)
        {
            if((details[i][11]-'0')*10 + (details[i][12]-'0') > 60)
            {
                count++;
            }
        }
        return count;    
    }
};

File: 2727-number-of-senior-citizens/number-of-senior-citizens.py
class Solution(object):
    def countSeniors(self, details):
        count = 0
        for i in range(len(details)):
            temp = list(details[i])
            num = 10*(int(temp[-4])) + (int(temp[-3]))
            print(num)
            if(num > 60):
                count += 1
        return count

        